CLAUDE.md

```
@@ -8,10 +8,12 @@ complaints-mcp is a Model Context Protocol (MCP) server that allows AI agents to
 
 ## Build and Test Commands
 
+This project uses **Just** (justfile) as its modern task runner, providing better cross-platform support and more features than traditional Makefiles.
+
 ### Building
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)